packages/perseus/src/widgets/explanation/explanation.test.ts

```diff
@@ -265,6 +267,24 @@ describe("Explanation", function () {
         expect(changeMock).toHaveBeenCalledWith("foo", "bar", callbackMock);
     });
 
+    it("Should get prompt json which matches the state of the UI", async () => {
+        // Arrange
+        const {renderer} = renderQuestion(question1);
+        const widget = renderer.getWidgetInstance("explanation 1");
+        const questionOptions = question1.widgets["explanation 1"].options;
+
+        // Act
+        const json = widget?.getPromptJSON?.() as ExplanationPromptJSON;
+
+        // Assert
+        expect(json).toEqual({
+            type: WidgetType.EXPLANATION,
+            showPrompt: questionOptions.showPrompt,
+            hidePrompt: questionOptions.hidePrompt,
```

Comment:
You think y'all will need the text telling learners how to close the explanation popover?

Reply:
Hmmm, probably not now that I stop to mull it over.